docs/images/workflow.diag

```diff
@@ -0,0 +1,39 @@
+blockdiag {
+  idea [label = "1) your new idea"];
+  investigate [label = "2) check if issue /\npull request\nalready exists"];
+  fork [label = "3) fork repository"];
+  branch [label = "4) create a branch"];
+  request [label = "5) create\npull request"];
+  add [label = "6) add your\ncontribution"];
+  test [label = "7) lint and test\nyour contribution\nlocally"];
```

Comment:
I would consider this optional. In my opinion it's not mandatory to setup the dev environment locally. That's what we have the CI for.

Reply:
I labelled this step as optional and gave it a light grey colour: commit f41f927aeb1091487c9200d3784eef946122ae91.